### PR TITLE
Add species selector banner to the homepage

### PR DESCRIPTION
**Pull request by msmanoj**
Added the banner to the homepage that will be displayed when no species are selected.

Related JIRA ticket: 
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-136

### CODE REVIEW

File: src/ensembl/src/content/home/Home.tsx [hunk 1]
```diff
@@ -57,6 +59,19 @@ const Home: FunctionComponent<HomeProps> = (props: HomeProps) => {
...
   return (
     <div className={styles.home}>
+      {!props.totalSelectedSpecies && (
+        <>
+          <span className={styles.speciesSelectorBannerText}>
+            7 species now available
+          </span>
+          <Link
+            className={styles.speciesSelectorBannerLink}
+            to={'/app/species-selector'}
```
Comment (azangru) [inline src/ensembl/src/content/home/Home.tsx]: I have a general question about links. Or rather about paths. Should we have some central place where to store / build them? For example, a url-helper, which will have methods or functions that build the required path?

```
const speciesSelector = () => '/app/speicies-selector';
```
and then in a component file:

```
import * as urlFor from 'path-to-url-helper';

<Link to={urlFor.speciesSelector()} />
```

@imran-ebi , what's your opinion about this?

Comment (imran-ebi) [reply src/ensembl/src/content/home/Home.tsx]: @azangru It is a good idea. Alternatively, we can use constants too for this.

Comment (azangru) [reply src/ensembl/src/content/home/Home.tsx]: I don't like the idea of constants, because urls will often contain dynamic elements.

Comment (msmanoj) [reply src/ensembl/src/content/home/Home.tsx]: Are we creating the helper in this PR or leaving it to a separate PR once we have more links available?

Comment (azangru) [reply src/ensembl/src/content/home/Home.tsx]: Good question. We probably might as well do it as a separate PR; but the links are creeping in in various places, so I would rather we do it pretty soon.

Comment (imran-ebi) [reply src/ensembl/src/content/home/Home.tsx]: @msmanoj If you create this now, please do let me know. I too might be able to make use of this function for the bootstrap code.

Comment (msmanoj) [reply src/ensembl/src/content/home/Home.tsx]: @azangru @imran-ebi added the `urlHelper` file to the path `src/shared/helpers/urlHelper.ts`.

**Pull request merged**